### ISSUE TITLE
烧录example/smart_light例程，下载进去一直重复打印 [crt] chan 1.....这样的信息

### ISSUE THREAD

**Issue by YUGEEV5** [espressif/esp-ali-smartliving#8]
## 1. 开发环境

- 模组或开发板：esp32
- esp-ali-smartliving 版本：1.6
- ESP-IDF 版本：3.3

## 2. 问题描述

烧录example/smart_light例程，下载进去一直重复打印 [crt] chan 1.....这样的信息，

### 2.1 复现步骤

我第一次修改了分区表，把三元组烧录进去，配网成功过一次。后来我运行了 make erase_flash 擦除了flash,并且又烧录进去了三元组，发现又出现了跑历程的情况，我再次修改分区表 发现没用了，串口显示一直再重复打印  [crt] chan 1..(最下面有调试信息)，
是不是这个是一次性的，一旦配网成功，擦除flash,所有的一切都没了？？？

## 3. 调试 Logs

I (366) app main: IOTX_AWSS_ENABLE
I (366) app main: IOTX_AWSS_START
[inf] zconfig_init(284): zconfig_init
I (376) wifi:ic_enable_sniffer
I (376) awss: Open monitor mode
[crt] chan 1

[crt] chan 6

[crt] chan 11

[crt] chan 1

[crt] chan 2

[crt] chan 3

[crt] chan 4

[crt] chan 5

[crt] chan 6

[crt] chan 7

[crt] chan 8

[crt] chan 9

[crt] chan 10

[crt] chan 11

**Comment by yuanjianmin** [comment]
是的,配网成功后,会把配网信息写到flash里面,如果执行make erase_flash后,配网信息被擦除,就会重新进入配网状态.
